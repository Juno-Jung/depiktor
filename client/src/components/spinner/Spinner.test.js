import React from 'react';
import Spinner from './Spinner';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  render(<Spinner />);
});

it('renders correctly', () => {
  const tree = renderer.create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
