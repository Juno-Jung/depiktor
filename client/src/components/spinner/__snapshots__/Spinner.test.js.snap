// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="spinner"
>
  <div
    className="rect1"
  />
  <div
    className="rect2"
  />
  <div
    className="rect3"
  />
  <div
    className="rect4"
  />
  <div
    className="rect5"
  />
</div>
`;
